UPCarouselFlowLayout/UPCarouselFlowLayout.swift

```diff
@@ -98,10 +99,17 @@ open class UPCarouselFlowLayout: UICollectionViewFlowLayout {
         
         let alpha = ratio * (1 - self.sideItemAlpha) + self.sideItemAlpha
         let scale = ratio * (1 - self.sideItemScale) + self.sideItemScale
+        let shift = ratio * (1 - self.sideItemShift) + self.sideItemShift
```

Comment:
No need for the `(1 - self.sideItemShift)` here as the shift is an absolute value an not a percentage. It actually makes the shift going from `sideItemShift` (e.g. `70`) to `1` when the item is centered (should be `0`).
This line should be: 
```
let shift = (1 - ratio) * self.sideItemShift
```